test/integration/typeful/cpp/objectbox-model.backup.json

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
Why was this file committed? Am I missing something?

Reply:
by mistake, removed